assets/src/dashboard/constants/index.js

```diff
@@ -91,10 +92,12 @@ export const secondaryPaths = [
   {
     value: APP_ROUTES.EDITOR_SETTINGS,
     label: __('Editor Settings', 'web-stories'),
+    inProgress: true,
   },
   {
     value: APP_ROUTES.SUPPORT,
     label: __('Support', 'web-stories'),
+    inProgress: true,
```

Comment:
I like this, we can flip routes on without having to mess with the flags, noice!